index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet"
+            href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="style.css">
+        <title>Edwin | Portfolio</title>
+    </head>
+
+    <body>
+        <header>
+            <section class="toolbar">
+                <a href="/" class="logo-container">
+                    <h3 class="logo-txt">Edwin G.</h3>
+                </a>
+                <nav class="menu-btn">
+                    <img src="./imgs/Union.svg" alt="menu icon">
+                </nav>
+            </section>
+        </header>
+        <section class="headline">
+            <h1 class="primary">
+                Hey There. I'm Edwin
+            </h1>
+            <h2 class="supporting">
+                I’m a Software Developer
+            </h2>
+            <p class="description">
+                I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you
+                see and have a project you need coded,
+                don’t hesitate to contact me.
+            </p>
+            <span class="socials">
+                <ul>
+                    <li><a href=""><i class="fa fa-github"></i></a></li>
+                    <li><a href=""><i class="fa fa-linkedin"></i></a></li>
+                    <li><a href=""><i class="fa fa-angellist"></i></a></li>
+                    <li><a href=""><i class="fa fa-twitter"></i></a></li>
+                    <li><a href=""><i class="fa fa-medium"></i></a></li>
+                </ul>
+            </span>
+        </section>
+    </body>
```

Comment:
- [ ] You should stick to the design as much as possible. please fix the spacing and also I would suggest you please download the social media icons from Figma and use it
### Your design
![Annotation 2023-01-12 191533](https://user-images.githubusercontent.com/81727988/212080079-e5d9925a-2662-4f71-8213-0b2909abee33.jpg)

### Figma design

![fimga](https://user-images.githubusercontent.com/81727988/212080347-7994acdb-f892-40b1-aebe-7e6e78a126ef.jpg)